deploy-scripts/mainnet-rsk/12_deploy_time_lock.ts

```diff
@@ -4,12 +4,17 @@ import type { DeployFunction } from "hardhat-deploy/types";
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   console.log("starting deployment: Timelock");
 
+  const day = 3600 * 24;
+
   const deployer = (await hre.getNamedAccounts()).DEPLOYER;
   const timelock = await hre.deployments.deploy("Timelock", {
     from: deployer,
     args: [
       deployer,
-      60 * 60 * 24 * 14 // 2 weeks delay
+      deployer,
+      day * 1, // min delay
+      day * 30, // max delay
+      day * 3 // grace period
```

Comment:
@tkeidar I'm changing it to give us a bit more than 3 days to execute the transaction after its ETA.
Let's say 1 week.
Please 👍  so @AndrzejGisWideLab and I would know it's ok to merge.
```suggestion
      day * 7 // grace period
```